docs/specs/servers/push/spec.md

```diff
@@ -58,17 +58,28 @@ OK, push-server v0.1.0
 
 ```
 POST <PUSH_SERVER_URL>/clients
+```
+
+Request body:
+```typescript
 {
-    "client_id": <CLIENT_ID>,
-    "type": <TYPE>,
-    "token": <DEVICE_TOKEN>
+  // The Client's ID from the Relay pairing.
+  client_id: string,
+
+  // The push service to use.
+  type: "apns" | "apns-sandbox" | "fcm" | "noop",
+
+  // The device's token for the push service.
+  token: string,
+
+  // See below
+  always_raw?: boolean,
```

Comment:
since `false` and `not present` are the same, why not make this non-nullable and default to false?

Reply:
Defaulting to true would mean that all clients support decrypted notifications which is not true. Clients to support this feature would have to set this flag to true on the SDK level and also set up a background service